src/TerraformPlanResultsPR.groovy

```diff
@@ -44,10 +44,12 @@ class TerraformPlanResultsPR implements TerraformPlanCommandPlugin, TerraformEnv
     @Override
     public void apply(TerraformPlanCommand command) {
         if (landscape) {
```

Comment:
Building landscape logic into this plugin is continuing to feel "wrong".  This PR is already uncovering that there's a weakness in TerraformCommands, and it's starting to address the weakness (by pushing `-out=tfplan` into an argument, where it belongs).  I think we need to take this further, and maybe start spelling out a `.withOutputRedirection` or something similar, for `2>plan.err` to live.  

If the stderr redirection was treated differently than a normal "suffix" (and as you're illustrating, it is different), then we could push the landscape logic out of this plugin as well.

^-- this doesn't need to be done as part of this particular PR, but it should be done before the PR Plugin is released.